Adding my project writeup

jelockro opened the pull request:
Hello, here is my writeup for completing the Deploy Native Build & Release Pipelines with Tekton and ArgoCDs

jelockrow committed:
adding jelockros project writeup

jelockrow committed:
adding jelockros challenge write-up

jelockrow committed:
adding jelockro with updated information

OliverMensahDev reviewed (changes requested):
Awesome work @jelockro 

It seems you didn't include those YAML files in your GitHub repository. 
Kindly share the project you work on in the repository. 
Thank you.

jelockro commented:
Thanks @OliverMensahDev
I have updated my main branch of https://github.com/jelockro/digital-ocean-k8s-challenge to include the submodules: https://github.com/jelockro/tekton-pipeline-example-app.git and https://github.com/jelockro/tekton-pipeline-example-pipeline.git.  
I also added the service files for argocd and ambassador and the secret files with credentials edited out.
